packages/legacy/core/App/modules/history/context/historyManager.tsx

```diff
@@ -0,0 +1,245 @@
+/**
+ *  USAGE
+ * 
+        const { agent } = useAgent()
+
+        //Log history
+        const historyManager = container.resolve(TOKENS.FN_LOAD_HISTORY)(newAgent)
```

Comment:
where is `newAgent` coming from in this usage example? Do you mean just `agent`?

Reply:
yes I meant "agent", fixed the typo